asdf_astropy/converters/unit/equivalency.py

```diff
@@ -0,0 +1,24 @@
+from asdf.extension import Converter
+
+
+class EquivalencyConverter(Converter):
+    tags = ["tag:astropy.org:astropy/units/equivalency-*"]
+
+    types = ["astropy.units.equivalencies.Equivalency"]
+
+    def to_yaml_tree(self, obj, tag, ctx):
+        return [
+            {'name': name, 'kwargs_names': list(kw.keys()), 'kwargs_values': list(kw.values())}
+            for name, kw in zip(obj.name, obj.kwargs)
+        ]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from astropy.units import equivalencies
+
+        components = []
+        for equivalency_node in node:
+            equivalency_method = getattr(equivalencies, equivalency_node["name"])
+            kwargs = dict(zip(equivalency_node["kwargs_names"], equivalency_node["kwargs_values"]))
+            components.append(equivalency_method(**kwargs))
+
+        return sum(components[1:], components[0])
```

Comment:
What does the `sum` do here?  A comment would be nice.

Reply:
It concatenates the lists together -- it's done this way to make sure we get an Equivalency object at the end instead of a vanilla list.  I'll add a comment.